dataland-frontend/tests/component/components/resources/dataTable/MultiLayerDataTable.cy.ts

```diff
@@ -158,7 +157,7 @@ describe("Tests for the MultiLayerDataTable component", () => {
       getCellContainer("Level 3 - String").should("be.visible");
 
       getSectionHead("Section 1").should("have.attr", "data-section-expanded", "true").click();
-      getSectionHead("Subsection 1").should("not.be.visible");
+      getSectionHead("Subsection 1", false).should("not.be.visible");
```

Comment:
Here the repeating commands to check the section heads are moved to a function to get leaner code, correct?